src/Android/Services/BiometricService.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Android.Content;
+using Android.Security.Keystore;
+using Bit.Core.Abstractions;
+using Java.Security;
+using Javax.Crypto;
+
+namespace Bit.Droid.Services
+{
+    class BiometricService : IBiometricService
+    {
+        static readonly string KEY_NAME = "com.8bit.bitwarden.biometric_integrity";
+
+        static readonly string KEYSTORE_NAME = "AndroidKeyStore";
+
+        static readonly string KEY_ALGORITHM = KeyProperties.KeyAlgorithmAes;
+        static readonly string BLOCK_MODE = KeyProperties.BlockModeCbc;
+        static readonly string ENCRYPTION_PADDING = KeyProperties.EncryptionPaddingPkcs7;
+        static readonly string TRANSFORMATION = KEY_ALGORITHM + "/" + BLOCK_MODE + "/" + ENCRYPTION_PADDING;
+
+        readonly KeyStore _keystore;
+
+        Context context = Android.App.Application.Context;
```

Comment:
Is this used anywhere? Not sure if this will have any issues or not, but we use this `CrossCurrentActivity.Current.Activity` plugin for getting context in other services. `CrossCurrentActivity.Current.Activity.Application.Context` should work.

Reply:
The Context was previously used when I experimented with things, will remove it.